test/OpenTelemetry.Tests/Trace/TracerProviderSdkTest.cs

```diff
@@ -1039,6 +1041,68 @@ public void SdkPopulatesSamplingParamsCorrectlyForLegacyActivityWithInProcParent
         activity.Stop();
     }
 
+    [Theory]
+    [InlineData(null, null, "ParentBased{AlwaysOnSampler}")]
+    [InlineData("always_on", null, "AlwaysOnSampler")]
+    [InlineData("always_off", null, "AlwaysOffSampler")]
+    [InlineData("always_OFF", null, "AlwaysOffSampler")]
+    [InlineData("traceidratio", "0.5", "TraceIdRatioBasedSampler{0.500000}")]
+    [InlineData("traceidratio", "not_a_double", "TraceIdRatioBasedSampler{1.000000}")]
+    [InlineData("parentbased_always_on", null, "ParentBased{AlwaysOnSampler}")]
+    [InlineData("parentbased_always_off", null, "ParentBased{AlwaysOffSampler}")]
+    [InlineData("parentbased_traceidratio", "0.111", "ParentBased{TraceIdRatioBasedSampler{0.111000}}")]
```

Comment:
Add a test case for "parentbased_traceidratio" with "not_a_double" as the arg value.